presentations/2024-01-25_coffee-and-coding/index.qmd

```diff
@@ -0,0 +1,149 @@
+---
+title: "Coffee and Coding"
+subtitle: "{targets}"
+author: "[Jacqueline Grout](mailto:jacqueline.grout1@nhs.net) and [Matt Dray](mailto:matt.dray@nhs.net)"
+date: 2024-01-25
+date-format: "MMM D, YYYY"
+knitr:
+  opts_chunk: 
+    eval: false
+    echo: true
+format:
+  revealjs:
+    theme: [default, ../su_presentation.scss]
+    transition: none
+    chalkboard:
+      buttons: false
+    preview-links: auto
+    slide-number: false
+    auto-animate: true
+    footer: |
+      view slides at [the-strategy-unit.github.io/data_science/presentations][ds_presentations]
+    width: 1920
+    height: 1080
+editor: 
+  markdown: 
+    wrap: 120
+---
+
+## `{targets}` for analysts
+
+::: columns
+::: {.column width="50%"}
+> Tom previously presented about {targets} at a coffee and coding last March and you can revisit his presentation and learn about the reasons why you should use the package to manage your pipeline and see a simple demonstration of how to use the package.
```

Comment:
Might be preferable to remove the Markdown quote marker (`>`) throughout, unless the line is a quote.

Reply:
It seems to squash the chunks of text up when I remove the quote markers and I don t like the way it makes it look. Any ideas how to make quarto lay it out nicely without the quote marker?

Reply:
Ah yes, the separation is quite small. Perhaps each chunk could be a bullet to help delineate?